src/main/java/com/codessquad/qna/domain/Answer.java

```diff
@@ -0,0 +1,63 @@
+package com.codessquad.qna.domain;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Entity
+public class Answer {
+    private static final DateTimeFormatter pattern = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
대문자로 변수명이 붙여져야 하지 않을까요? 
그리고 무엇을 위한 패턴인지가 변수명을 통해 드러나면 좋겠습니다.

```suggestion
    private static final DateTimeFormatter PATTERN = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");

Reply:
FORMATTER_PATTERN 으로 변수명 변경하였습니다.